playbooks/init/base_packages.yml

```diff
@@ -39,7 +39,7 @@
       - "{{ 'python-ipaddress' if ansible_distribution != 'Fedora' else '' }}"
       - libsemanage-python
       - yum-utils
-      - "{{ 'python3-docker' if ansible_distribution == 'Fedora' else 'python-docker' }}"
+      - "{{ 'python3-docker' if ansible_distribution == 'Fedora' else 'python-docker-py' }}"
```

Comment:
Actually I am unsure about this one. Looks a lot like scope creep but was a nessesery change to get my test environment working.

Reply:
Actually once I rebased to upstream, this changed vanished. It was already changed there but we did not pull yet. I will remove it from this PR.